python/ray/exceptions.py

```diff
@@ -130,8 +130,19 @@ def __init__(
         self.traceback_str = traceback_str
         self.actor_repr = actor_repr
         self._actor_id = actor_id
-        # TODO(edoakes): should we handle non-serializable exception objects?
         self.cause = cause
+
```

Comment:
Could you update the docs to mention this behavior?

See #44379 as an example.